nss-day-cohort-48/giffygram-jumpstation
# Added date tracking to posts.

#### Changes Made

1. Modified file PostList.js to include date tracking and display functionality.
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout nrh-adding-post-data
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   Make a post and confirm that a date pops up in the post
4. View code file.
   Witness the fresh spaghetti
